component/src/main/java/me/androidbox/component/event/VisitorItem.kt

```diff
@@ -0,0 +1,104 @@
+package me.androidbox.component.event
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.R
+import me.androidbox.component.ui.theme.*
+
+@Composable
+fun VisitorItem(
+    visitorInitials: String,
+    visitorName: String,
+    isCreator: Boolean = true,
+    modifier: Modifier = Modifier,
+    onDeleteClicked: (visitorName: String) -> Unit) {
+
+    Row(modifier = modifier
+        .fillMaxWidth()
+        .background(color = MaterialTheme.colorScheme.visitorBackgroundColor),
+        verticalAlignment = Alignment.CenterVertically) {
+        Text(
+            modifier = Modifier
+                .padding(12.dp)
+                .drawBehind {
+                    drawCircle(
+                        color = Color.LightGray,
+                        radius = this.size.maxDimension
+                    )
+                },
```

Comment:
No need for a canvas here, you can simply clip the content:
```
Modifier.clip(CircleShape).background(Color.LightGray)
```

Reply:
@philipplackner Trying this here:
```
        Text(
            modifier = Modifier
                .clip(CircleShape)
                .background(Color.LightGray),
            text = visitorInitials,
            fontSize = 12.sp,
            color = MaterialTheme.colorScheme.visitorInitialsFontColor,
            fontWeight = FontWeight.SemiBold
        )
```
![Screenshot from 2023-03-29 20-37-12](https://user-images.githubusercontent.com/14260802/228555786-0912b059-df26-49e4-b03c-5289966fd066.png)